src/main/java/io/vertx/proton/ProtonServerOptions.java

```diff
@@ -266,23 +267,53 @@ public ProtonServerOptions setSni(boolean sni) {
   }
 
   /**
-   * Set the heartbeat (in milliseconds) as maximum delay between sending frames for the remote peers.
-   * If no frames are received within 2*heartbeat, the connection is closed
+   * Sets the heart beat (in milliseconds) as maximum delay between sending frames for the remote peers.
+   * If no frames are received within 2 * heart beat, the connection is closed.
    *
-   * @param heartbeat hearthbeat maximum delay
-   * @return  current ProtonServerOptions instance
+   * @param heartbeat heart beat maximum delay
+   * @return current ProtonServerOptions instance
    */
   public ProtonServerOptions setHeartbeat(int heartbeat) {
     this.heartbeat = heartbeat;
     return this;
   }
 
   /**
-   * Return the heartbeat (in milliseconds) as maximum delay between sending frames for the remote peers.
+   * Gets the heart beat (in milliseconds) as maximum delay between sending frames for the remote peers.
    *
-   * @return  hearthbeat maximum delay
+   * @return heart beat maximum delay
    */
   public int getHeartbeat() {
     return this.heartbeat;
   }
+
+  /**
+   * Sets the maximum frame size to announce in the AMQP <em>OPEN</em> frame.
+   * <p>
+   * If this property is not set explicitly, a reasonable default value is used.
+   * <p>
+   * Setting this property to -1 will result in no maximum frame size being announced at all.
```

Comment:
Perhaps say "negative" rather than "to -1" to match what it appears the impl does?